src/main/java/org/kizombadev/markdownparser/SyntaxAnalyzer.java

```diff
@@ -34,6 +61,8 @@ public Syntax parse(ImmutableList<Token> tokens) {
                 handleQuotation(root);
             } else if (currentToken.equals(Token.DoubleStar)) {
                 handleBold(root);
+            } else if (currentToken.equals(Token.Star) && nextToken() == Token.Blank) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Use the "equals" method if value comparison was intended. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1698)